### PR TITLE
Remove impractical rule

### PR DESCRIPTION
**Pull request by david-poindexter**
## Summary
@dnnsoftware/approvers I'm thinking this is a bit overkill, as I can't think of a reason why it would matter who actually clicks the merge button.  The important thing is that there are two approvals.  Do you all agree?

### CODE REVIEW

**Review by valadas — APPROVED** [review]
Yes I approve, very often we have 2 approvals and need to wait quite some time to keep working on next steps while waiting just for CI. And well the persons who can actually click that button are also approvers. So I think this makes sense.

**Review by mitchelsellers — APPROVED** [review]

**Pull request merged**
